TaskArray/TaskArray5/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace TaskArray5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Input number of values in an array");
+            var n = Convert.ToInt32(Console.ReadLine());
+            int[] arr = new int[n];
+            for (int a = 0; a < arr.Length; a++)
+            {
+                arr[a] = Convert.ToInt32(Console.ReadLine());
+            }
+            int consilience = 0;
+            for (int a = 0; a < arr.Length; a++)
+            {
+                for (int i = 0; i < arr.Length; i++)
```

Comment:
можно цикл начинать с переменной а, тогда не нужно будет a == i проверять
```suggestion
                for (int i = a + 1; i < arr.Length; i++)
```

Reply:
![image](https://user-images.githubusercontent.com/81630223/114318529-104d0180-9b16-11eb-8ca7-929db5372c08.png)

Reply:
программа не будет сравнивать с уже пройденными цифрами 
 
![image](https://user-images.githubusercontent.com/81630223/114318599-65891300-9b16-11eb-93f5-ce1203ee6735.png)

Reply:
я все таки свой вариант оставлю =)